versions.tf

```
@@ -16,7 +16,8 @@
 
 terraform {
   required_version = ">= 0.12"
+
   required_providers {
-    google = "~> 2.12"
+    google = "~> 3.5.0"
```

For the module itself, we should pin to a major version:

```suggestion
    google = "~> 3.5"
```